exercises/exercise-2/exercise-2.js

```diff
@@ -70,3 +70,22 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+//Task 1
+hogwarts.filter(obj => {
+let {house} = obj;
```

Comment:
you can deconstruct these in the param, same for the other functions

```suggestion
hogwarts.filter(({house}) => {
```